web/tests/test_pandas_web.py

```diff
@@ -0,0 +1,22 @@
+import yaml
+import os
+# Not working in code. the pandas directory is set up for this, but not the web directory
+from pandas_web import Preprocessors
+import pytest
+
+@pytest.fixture # I think this is fine
```

Comment:
Fixtures are functions to be reused in different tests. Here you're decorating a test itself, which doesn't seem to make sense.